app/common/helpers/collections.py

```diff
@@ -405,13 +405,26 @@ def toggle_form_completed(self, form: "Form", user: "User", is_complete: bool) -
                 self.submission, SubmissionEventKey.FORM_RUNNER_FORM_COMPLETED, form
             )
 
-    def get_next_question(self, current_question_id: UUID) -> Optional["Question"]:
+    # todo: decide if the add another index should be available submission helper wide where it just checks self
+    #       does having it on lots of methods increase the cognitive load/ complexity
```

Comment:
I actually think having it here in the signature reduces complexity as it's very clearly telling this method which one to use